src/OrchardCore.Modules/OrchardCore.Search.AzureAI/Drivers/AzureAISearchDefaultSettingsDisplayDriver.cs

```diff
@@ -58,9 +59,16 @@ public override IDisplayResult Edit(ISite site, AzureAISearchDefaultSettings set
         {
             model.AuthenticationTypes =
             [
-                new SelectListItem(S["Default"], nameof(AzureAIAuthenticationType.Default)),
-                new SelectListItem(S["Managed Identity"], nameof(AzureAIAuthenticationType.ManagedIdentity)),
-                new SelectListItem(S["API Key"], nameof(AzureAIAuthenticationType.ApiKey)),
+                new SelectListItem(S["Default"], nameof(AzureAuthenticationType.Default)),
```

Comment:
I thought the Azure modules is supposed to abstract it. Also should this really be in the admin?

Reply:
We already have Default on as an option. I added the result of the supported types